tests/Maintenance/MaintenanceTest.cpp

```diff
@@ -715,38 +951,405 @@ TEST_F(MaintenanceTestActionPhaseOne,
   }
 }
 
+
+TEST_F(MaintenanceTestActionPhaseOne, leader_behaviour_plan_self_current_self) {
```

Comment:
In this test suite, `current` is consistently used for `local`, which I find confusing. PhaseOne has nothing to do with Current. We should rename this.

Reply:
Done!